templates/lxd/index.html

```diff
@@ -0,0 +1,299 @@
+            alt="",
+            width="151",
+            height="84",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <div class="col-3 col-medium-2">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/4c69c936-Edge%20Cloud%201.png",
+            alt="",
+            width="168",
+            height="112",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+        <div class="col-3 col-medium-2">
+          {{ image (
+            url="https://assets.ubuntu.com/v1/b47f5f82-shields-security%201.png",
+            alt="",
+            width="151",
+            height="79",
+            hi_def=True,
+            loading="lazy"
+            ) | safe
+          }}
+        </div>
+      </div>
+      <hr class="p-rule u-hide--small">
+      <div class="row p-section--shallow">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">Run a full Linux OS inside a container</h3>
+          <p>LXD system containers run a complete filesystem with background processes. This allows you to run any workload, or containerise your traditional systems and apps without modifying the apps or your operations. LXD containers offer the density and efficiency of containers with a VM-like experience.</p>
+        </div>
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">Cloud-like experience</h3>
+          <p>LXD is image based and supports images for a large number of Linux distributions. It comes with built-in image stores that supply official Ubuntu images and images provided by the community, which allows you to spin up a container or a VM in a matter of seconds. Existing integrations with various deployment and orchestration tools allow you to manage your infrastructure in a cloud-like way.</p>
+        </div>
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">Secure and scalable</h3>
+          <p>LXD runs unprivileged containers by default &mdash; protecting the host system from potential attacks. For virtual machines, LXD uses modern virtual hardware (VirtIO) exclusively. In addition, it utilises UEFI SecureBoot and provides vTPM support.</p>
+          <p>Resource restrictions through cgroups and ulimits, as well as RBAC, are also supported.</p>
+          <p>LXD is also easy to scale &mdash; from containers on your laptop to thousands of instances in the data centre.</p>
+        </div>
+      </div>
+    </div>
+  </div>
+</section>
+
+<section class="p-section">
+  <div class="u-fixed-width p-section--shallow">
+		<hr class="p-rule">
+		<h2>Alternatives for running workloads</h2>
+	</div>
+  <div class="row--25-75">
+    <div class="col">
+      <hr class="p-rule u-hide--medium u-hide--small">
+      <div class="row">
+        <div class="col-3 col-medium-2">
+          <h3 class="p-heading--5">System containers</h3>
```

Comment:
These have bold hrs above them on the design:
![image](https://github.com/canonical/canonical.com/assets/17607612/179e1be0-d0e0-4162-93f9-fa185c84c900)

Reply:
fixed